tests/CMakeLists.txt

```diff
@@ -85,3 +85,6 @@ add_subdirectory(concepts)
 
 ### TESTS : impl ###
 add_subdirectory(impl)
+
+### TESTS : view ###
+add_subdirectory(view)
```

Comment:
In Kokkos, the directory name starts with an uppercase letter: `core/src/View/...`. Should we use the uppercase letter too for `array` and `view` subdirectories that you introduce in this PR?

Reply:
I would argue that the case of the folder should match the case of the namespace.

Reply:
OK! It's fine either way.